packages/adp-tooling/test/unit/prompts/add-component-usages/indest.test.ts

```diff
@@ -0,0 +1,314 @@
+import * as i18n from '../../../../src/i18n';
```

Comment:
Seems like a type in the file name.

Reply:
Fixed in: bdd8262301bee934e0dcfd5def4dd5d18267b793